doc/v2/documentation_data_format.rst

```diff
@@ -644,20 +642,36 @@ Detailed field description
   observable formula contains an override, and a proportional noise model is
   used, which means the observable formula also appears in the noise formula.
 
-- ``noiseDistribution`` [STRING: 'normal' or 'laplace', OPTIONAL]
+* ``noiseDistribution`` [STRING, OPTIONAL]
+
+  Assumed noise distribution for the measurements of the given observable.
+
+  Supported options are:
 
-  Assumed noise distribution for the given measurement. Only normally or
-  Laplace distributed noise is currently allowed (log-normal and
-  log-Laplace are obtained by setting ``observableTransformation`` to ``log``, similarly for ``log10``).
-  Defaults to ``normal``. If ``normal``, the specified ``noiseParameters`` will be
-  interpreted as standard deviation (*not* variance). If ``Laplace`` ist specified, the specified ``noiseParameter`` will be interpreted as the scale, or diversity, parameter.
+  * ``normal``: Gaussian noise model with standard deviation as specified in
+    ``noiseFormula``.
+  * ``log-normal``: Log-normal noise model with standard deviation as specified
+    in ``noiseFormula``. I.e., the logarithm of the measurements is normally
+    distributed.
+  * ``laplace``: Laplace noise model with scale parameter as specified in
+    ``noiseFormula``.
+  * ``log-laplace``: Log-Laplace noise model with scale parameter as specified
```

Comment:
Rename to match ``parameter.priorDistribution`` (https://github.com/PEtab-dev/PEtab/pull/595)